templates/account/signup.html

```diff
@@ -85,4 +105,41 @@ <h2 class="text-xl text-center">Or Sign Up with Email</h2>
 
   </div>
 
+  <script>
+
+    function toggleButtonsOff(state) {
+      let gitButton = document.getElementById("gitButton");
+      let googleButton = document.getElementById("googleButton");
+      let signupButton = document.getElementById("signup");
+      if (state) {
+        gitButton.classList.add("disabledLink");
+        googleButton.classList.add("disabledLink");
+        signupButton.classList.add("disabledLink");
+        return
+      }
+        gitButton.classList.remove("disabledLink");
+        googleButton.classList.remove("disabledLink");
+        signupButton.classList.remove("disabledLink");
+      return
+    }
+
+    function accepted() {
+      let tosCheckbox = document.getElementById("readTOS");
+      if (tosCheckbox.checked) {
+        toggleButtonsOff(false)
+      } else {
+        toggleButtonsOff(true)
+      }
+    }
+
+window.addEventListener('load', function() {
+  let nojs = document.getElementById("jsfallback");
+  let yesjs = document.getElementById("jsvalidation");
+    nojs.classList.add("hidden");
+    yesjs.classList.remove("hidden");
+    toggleButtonsOff(true)
```

Comment:
```
const tosCheckbox = document.getElementById('readTOS');
tosCheckbox.addEventListener('change', handleTosChange);
```
**suggestion:** After this line, we could add an event listener to the checkbox directly, eliminating the need for an explicit call in the markup
**todo:** Fix indentation

Reply:
not needed, prefer the onclick.